src/components/Wallet.vue

```diff
@@ -11,12 +24,133 @@ export default {
   components: {},
   data: function () {
     return {
-      btc: 0,
+      positions: {
+        clientX: undefined,
+        clientY: undefined,
+        movementX: 0,
+        movementY: 0,
+      },
+      result: "",
+      coinId: "",
     };
   },
+  methods: {
+    fetchAPIData() {
+      fetch(`https://coinranking1.p.rapidapi.com/coin/${this.coinId}`, {
+        method: "GET",
+        headers: {
+          "x-rapidapi-key":
+            "6a74bac8a4mshd570be2510616ebp18000ajsne44b784d3645",
+          "x-rapidapi-host": "coinranking1.p.rapidapi.com",
+        },
+      })
+        .then((response) => response.json())
+        .then((json) => {
+          this.result = `The coin you choosed is ${
+            json.data.coin.name
+          } with symbol ${
+            json.data.coin.symbol
+          } and actual price ${json.data.coin.price.slice(0, 7)}$`;
```

Comment:
Unikałbym działania na danych pobieranych z api w response. Zamiast tego proponuję dodać zmienną np. `coin`.
`this.coin = json.data.coin;`

A ten fragment można albo umieścić bezpośrednio w template: 
```
The coin you choosed is {{  coin.name }} with symbol {{ coin.symbol }} and actual price {{ coin.price.slice(0, 7) }}$
```
lub jako computed:
```
computed: {
   result() {
      return `The coin you choosed is ${ this.coin.name } with symbol ${ this.coin.symbol } and actual price ${ this.coin.price.slice(0, 7) }$`;
   }
}
```

Fajnie, że uzywasz template string. To teraz standard i dobry wzorzec.